ci-operator/step-registry/ipi/install/install/in-bastion/ipi-install-install-in-bastion-commands.sh

```diff
@@ -231,10 +248,15 @@ ls -ltra ${dir}
 echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_END"
 date "+%F %X" > "${SHARED_DIR}/CLUSTER_INSTALL_END_TIME"
 
+# clean up the ${REMOTE_DIR} as it may contains credential files
+run_ssh_cmd "${SSH_PRIV_KEY_PATH}" "${BASTION_SSH_USER}" "${BASTION_IP}" "rm -rf ${REMOTE_DIR}/*"
```

Comment:
I guess we can move the cleanup to `prepare_next_steps` function, so that the cleanup always happen.